src/main/java/util/DataConverterImpl.java

```diff
@@ -0,0 +1,26 @@
+package util;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.FruitTransaction;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> lines) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        for (String line : lines) {
+            String[] parts = line.split(",");
+            FruitTransaction.Operation operation = FruitTransaction.Operation.fromCode(parts[0]);
+            String fruit = parts[1];
+            int quantity = Integer.parseInt(parts[2]);
+
+            FruitTransaction transaction = new FruitTransaction();
+            transaction.setOperation(operation);
+            transaction.setFruit(fruit);
+            transaction.setQuantity(quantity);
+
+            transactions.add(transaction);
+        }
+        return transactions;
+    }
+}
```

Comment:
The class `DataConverterImpl` should be placed in a package that reflects its purpose. According to the checklist, the project structure should have a `service` package for holding services and a `service.impl` for holding implementations of services. Please move this class to the appropriate package.